src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -56,17 +79,32 @@
     @DeleteMapping("/user/{id}")
     @Operation(summary = "Удаление пользователя")
     fun delete(@CookieValue("jwt") jwt: String?, @PathVariable id: Int): ResponseEntity<Any> {
+
+        if (!ftService.isEnabled(userCanDelete)) {
+            println("feature $userCanDelete is off")
+            return ResponseEntity.ok().body(Message("FT $userCanDelete is off"))
+        }
+        println("feature $userCanDelete is on")
+
         return userService.deleteUser(id)
     }
 
     @GetMapping("/user")
     @Operation(summary = "Получение данных о пользователе")
     fun getUser(@CookieValue("jwt") jwt: String?): ResponseEntity<Any> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty